app/templates/browser-support.hbs

```diff
@@ -0,0 +1,151 @@
+{{page-title "Browser Support"}}
+<div class="container layout">
+  <section aria-labelledby="section-browser-support-policy">
+    <h1 id="section-browser-support-policy">Ember.js Browser Support Policy</h1>
+
+    <p>
+      Ember supports the following major browsers:
+    </p>
+
+    <ul class="list-unstyled layout my-3">
+      <EsCard class="lg:col-2">
+        <h3 class="text-center">Desktop</h3>
```

Comment:
I think using `<h3>` right after `<h1>` will fail accessibility audit. Can we use a different tag? (We can always customize styling.)